README.md

```diff
@@ -51,6 +51,9 @@ v install https://github.com/webui-dev/v-webui
 ```v
 import vwebui as ui
 
+
+
+
```

Comment:
To make it into a successful contribution, can you please keep the code in the example formatted without adding additional line breaks and by adding indentation to the main function block.

Reply:
Sure!

Reply:
Did it!

Reply:
Thanks for the contribution!